_automation_cookbook/ternary-operator.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Ternary Operator
+excerpt: Use ternary operators for conditional logic in Cerb automations.
```

Comment:
We can leave off the "in Cerb automations". This text is used on the automation cookbook index, so mentioning Cerb automations is redundant.